Tests/DependencyInjection/Fixtures/config/xml/dbal_service_multiple_connections.xml

```diff
@@ -21,7 +21,9 @@
                 user="sqlite_user"
                 password="sqlite_s3cr3t"
                 path="/tmp/db.sqlite"
-                memory="true" />
+                memory="true">
+                <option key="userDefinedFunctions" />
```

Comment:
The XSD currently doesn't allow defining anything other than a scalar value for an option. I'm looking at how to fix this.

Reply:
I was able to fix this with some help from @dkarlovi and @wouterj. Thanks to both!